test/pytest_coreneuron/test_coreneuron_configuration.py

```diff
@@ -0,0 +1,24 @@
+from neuron import coreneuron
+
+
+def test_coreneuron_configuration():
+    with coreneuron(
+        restore_path="restore",
+        save_path="save",
+        model_path="coredat",
+        skip_write_model_to_disk=True,
+    ):
```

Comment:
I have modified my test that changing/testing coreneuron parameters in a context so that it won't change the coreneuron setting used by other unit tests. Previously the failure we saw in other tests after adding my test are likely due to this. I could also manually revert the values but it seems cleaner with the context manager.

Reply:
@pramodk , this PR is ready. If it looks good to you, could you please merge it ? Thanks!